.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,93 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["type: bug","triage me"]
+body:
+- type: markdown
+  attributes:
+    value: |
+        _**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response._
+- type: checkboxes
+  attributes:
+    label: Determine this is the right repository
+    description: Please follow the steps in [the SUPPORT guide](https://github.com/vchudnov-g/google-cloud-python/blob/issue-templates/SUPPORT.md) and determine that this is likely a client library issue and this is the correct repository in which to report it.
+    options:
+      - label: I determined this is the correct repository in which to report this bug
+        required: true
+- type: textarea
+  attributes:
+    label: Summary of the issue
+    description: Describe succinctly what you expected to happen, and what actually happened.
+    placeholder: Describe the issue you encountered
+    value: |
+        **Context**
+        _e.g. I was trying to use the new method Foo_
+    
+        **Expected Behavior:**
+        _e.g. I expected method Foo to paginate_
+
+        **Actual Behavior:**
+        _e.g. Method Foo did not paginate_        
+- type: input
+  attributes:
+    label: API client name and version
+    description: Try `pip list | grep google-cloud`, which should give you a name corresponding to one of the packages listed [here](https://github.com/googleapis/google-cloud-python/tree/main/packages).
```

Comment:
```suggestion
    description: Try `pip list | grep google`, which should give you a name corresponding to one of the packages listed [here](https://github.com/googleapis/google-cloud-python/tree/main/packages).
```

Reply:
Good idea. Did it with a trailing hyphen. WDYT?

Reply:
It may be helpful without the trailing hyphen as an additional package `googleapis-common-protos` will be included

Reply:
OK, good catch! Will do.